server/monitor/src/main/java/org/apache/accumulo/monitor/view/WebViews.java

```diff
@@ -47,12 +57,43 @@
 @Produces(MediaType.TEXT_HTML)
 public class WebViews {
 
+  private static final Logger log = LoggerFactory.getLogger(WebViews.class);
+
+  /**
+   * Get HTML for CSS and JS imports from configuration. See ACCUMULO-4739
+   *
+   * @param model map of the MVC model
+   */
+  private void addImports(Map<String,Object> model) {
+    AccumuloConfiguration conf = Monitor.getContext().getConfiguration();
+    String imports = conf.get(Property.MONITOR_HEADER_IMPORTS);
+    if (StringUtils.isEmpty(imports))
+      return;
+    List<String> monitorImports = new ArrayList<>();
+    ObjectMapper objectMapper = new ObjectMapper();
+    try {
+      for (MonitorImport monitorImport : objectMapper.readValue(imports, MonitorImport[].class)) {
```

Comment:
Could also map to `String[].class` instead. It could simplify a lot of this code and might provide a better balance between an arbitrary HTML blob, and a complex JSON object. Something like:

`[ "<script href=\"...\" integrity=\"...\" ... />", "<style src=\"...\" ... />" ]`

Reply:
+1 
This would satisfy the concerns from your comment above about flexibility as well.